### PR TITLE
JavaScript: Tag removal/status - all remaining

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: files/en-us/web/javascript/reference/deprecated_and_obsolete_features/index.md [hunk 1]
```diff
@@ -2,11 +2,8 @@
 title: Deprecated and obsolete features
 slug: Web/JavaScript/Reference/Deprecated_and_obsolete_features
 page-type: guide
-tags:
-  - Deprecated
-  - Guide
-  - JavaScript
-  - Deprecated
+status:
+  - deprecated
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/deprecated_and_obsolete_features/index.md:6]: I don't think this should have the deprecated status. It had the "deprecated" tag but that was weird as well (it even has two of it!)

Comment (hamishwillee) [inline files/en-us/web/javascript/reference/deprecated_and_obsolete_features/index.md:6]:
```suggestion
```

Comment (hamishwillee) [reply files/en-us/web/javascript/reference/deprecated_and_obsolete_features/index.md:6]: I wasn't sure it made sense either! Deleted by https://github.com/mdn/content/pull/24665/files#r1112412053

**Review by Josh-Cena — APPROVED** [review]
LGTM

**Pull request merged**